README.md

```diff
@@ -115,8 +109,30 @@ OP_RETURN
   [Signature]
 ```
 
-If a transaction includes a valid AIP signature and the signature involves all previous fields (AIP_ALL) it is not the sender but the signing address that will be considered the owner in the transaction. 
+If a transaction includes a valid AIP signature and the signature involves all previous fields (AIP_ALL) both the sender and AIP signing address will create an update to the key. Effectively this means that you will be able to access the content with both `D://<sender>/<key>` and `D://<AIP signing address>/<key>`.
+
 
+### Delete
+
+A key is "removed" by referencing a transaction with `NULL` as file content (`0x00`). The state of the planaria will provide the string `deleted` as "type" when a key is removed. 
```

Comment:
Great!

It may contradict this line? https://github.com/bitcoineler/D/pull/2/files#r270690587

Reply:
Solved.